streamlit: add page (#13143)
tldr-pages/tldr: pages/common/streamlit.md

# streamlit

> Framework for creating interactive, data-driven web apps in Python.
> More information: <https://docs.streamlit.io/>.

> - Check Streamlit installation:
> 
> `streamlit hello`
> 
> - Run your Streamlit application:
> 
> `streamlit run {{project_name}}`
> 
> - See all supported Streamlit commands:

Reviewer: ```suggestion
- Check for Streamlit installation:

`streamlit hello`

- Run your Streamlit application:

`streamlit run {{project_name}}`

- Display help:
```
For help and version commands we generally use this generic wording across our pages (for easy translations).

Author: Understood, rest of document seems in a generic format. so I'll make the given change only

`streamlit --help`

> - Check the current version of Streamlit:

Reviewer 2: ```suggestion
- Display version:
```
We generally use the same description for help and version command across all pages.

Author: Sure!
